xcode/configs/base-mac.xcconfig

```diff
@@ -1,3 +1,2 @@
-ARCHS = $(NATIVE_ARCH_ACTUAL)
-VALID_ARCHS = i386 x86_64
```

Comment:
This setting is deprecated. Also i386 and x86_64 are intel architectures so this would no longer be the right value.